### DISCUSSION TITLE
What about manga?

### DISCUSSION THREAD

**Discussion by vannessqn** [marisukukise/japReader#26] [General]
I know this is not OCR program just want to ask what program best used to read raw manga?

**Comment by marisukukise** [comment]
Please open a [discussion](https://github.com/marisukukise/japReader/discussions) if you want to ask questions and not report issues.

I don't really know what "the best" program would be, some OCR program of your choice, I personally use ShareX for OCR sometimes, but it's not really reliable for longer words and sentences, so I can't help

**Comment by vannessqn** [comment]
sorry lol this is the first time I clicked on the "discussion" tab, never heard or use it before :)

**Comment by marisukukise** [comment]
I found a nice way to read manga using [mokuro](https://github.com/kha-white/mokuro) (and optionally also [Mokuro2Pdf](https://github.com/Kartoffel0/Mokuro2Pdf))

It takes raw manga jpgs and produces a manga pdf with selectable text, so you can copy that text and have japReader to read it

![image](https://github.com/marisukukise/japReader/assets/109931938/d1c40aad-8be6-427b-88b2-2aa38079d679)
